lib/model/store.dart

```diff
@@ -392,10 +392,10 @@ class PerAccountStore extends ChangeNotifier {
     }
   }
 
-  Future<void> sendMessage({required MessageDestination destination, required String content}) {
+  Future<void> sendMessage({required MessageDestination destination, required String content, bool? readBySender}) {
```

Comment:
So in contrast to the API binding layer, this method on PerAccountStore I think actually is a good place to hard-code this parameter to true. I'll just add a commit on top making that change.

Here's the commit message I wrote for that commit, explaining why:

    send [nfc]: Always pass readBySender true
    
    I think we're always going to want this parameter to be true
    for all the messages we send from the app.
    
    In the API bindings themselves, i.e. in lib/api/ , I generally
    prefer not to hard-code this sort of thing.  That lets the API
    bindings be a faithful reflection of the Zulip Server API, with
    a minimum of ad-hoc specializations for the needs of our app.
    So that's why the `sendMessage` binding just takes the parameter
    from its caller.
    
    But this method PerAccountStore.sendMessage is already something
    specific to our app rather than an API binding; it doesn't do much
    now beyond wrap the API, but the reason it exists is to be the
    future home of our "outbox" logic.  That means (a) recording the
    message locally so that we can display it immediately, (b) possibly
    retrying the send, and (c) if the send fails or times out, recording
    that fact so that our UI can present the user with that information
    and the option to retry.
    
    So that makes it a natural home for encapsulating any other logic
    that this app will always want when sending a message.
    In particular, passing this parameter.